qa/os/src/test/java/org/elasticsearch/packaging/util/Distribution.java

```diff
@@ -92,6 +95,10 @@ public boolean isDocker() {
             this.extension = extension;
             this.compatible = compatible;
         }
+
+        public boolean isDocker() {
```

Comment:
This doesn't appear to be used, since the same condition is written in isDocker above?

Reply:
Ah, yeah, good catch.